lib/ads-adal-library/src/engine/AzureAuth.ts

```diff
@@ -135,14 +135,17 @@ export abstract class AzureAuth {
 				expiresOn = Number(result.expiresOn);
 			}
 			// Let's just return here.
-			if (accessToken) {
-				return {
-					...accessToken,
-					expiresOn: expiresOn,
-					tokenType: 'Bearer'
-				};
+			if (accessToken.expiresOn) {
```

Comment:
What scenario would we have an accessToken but not an expiration time?

Reply:
This is to check that the object exists (otherwise vscode was complaining):
![imagen](https://user-images.githubusercontent.com/5080892/181703485-895a02d0-ecc8-48fe-95c2-720363383f9e.png)